content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -187,7 +188,18 @@ replace it with `member4=http://10.0.0.4`.
    fd422379fda50e48, started, member3, http://10.0.0.3:2380, http://10.0.0.3:2379
    ```
 
-2. Remove the failed member:
+2. Do either of the following:
+
+   1. If each Kubernetes API server is configured to communicate with all etcd
+      members, remove the failed member from the `--etcd-servers` flag, then
+      restart each Kubernetes API server.
+   2. If each Kubernetes API server communicates with a single etcd member,
+      configure Kubernetes API server clients to not route requests to the
+      Kubernetes API server that communicates with the failed etcd member.
+      This can often be done by configuring a load balancer. Then stop the
+      Kubernetes API server that communicates with the failed etcd.
```

Comment:
I would simplify this, because if the apiserver is talking to a single failed etcd, then that apiserver is not going to be healthy to begin with, in which case, you already have broken clients. 
```suggestion
   2. If each Kubernetes API server communicates with a single etcd member,
      then stop the Kubernetes API server that communicates with the failed
      etcd.
    
```

Reply:
SGTM. Merging in.